test/utils/Processing.test.ts

```diff
@@ -0,0 +1,384 @@
+import fs from 'fs-extra';
+import path from 'path';
+import temp from 'temp';
+import { loggerSpy } from '../testhelpers/loggerSpy';
+import {
+  findInputDir,
+  ensureOutputDir,
+  readConfig,
+  loadExternalDependencies,
+  getRawFSHes,
+  writeFHIRResources,
+  getIgDataPath
+} from '../../src/utils/Processing';
+import * as loadModule from '../../src/fhirdefs/load';
+import { FHIRDefinitions } from '../../src/fhirdefs';
+import { Package } from '../../src/export';
+import { StructureDefinition, ValueSet, CodeSystem, InstanceDefinition } from '../../src/fhirtypes';
+import { PackageLoadError } from '../../src/errors';
+// let load = import load from '../../src/fhirdefs/load';
+// let fakeLoad = import {loadDependency} from '../../src/fhirdefs/load';
+
+describe('Processing', () => {
+  temp.track();
+
+  describe('#findInputDir()', () => {
+    let tempRoot: string;
+
+    beforeAll(() => {
+      tempRoot = temp.mkdirSync('sushi-test');
+      fs.mkdirSync(path.join(tempRoot, 'has-fsh'));
+      fs.mkdirSync(path.join(tempRoot, 'has-fsh', 'fsh'));
```

Comment:
You can use `ensureDirSync` or `mkdirpSync` and it will also create any parent folders for you.  So you wouldn't need code on line 30.